<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GerenciadorWindow</class>
 <widget class="QMainWindow" name="GerenciadorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GerenciadorWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>681</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listComandos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>661</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PID:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>451</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="textComando"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnComandos">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>451</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="textPID"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnParar">
         <property name="text">
          <string>Parar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnContinuar">
         <property name="text">
          <string>Continuar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMatar">
         <property name="text">
          <string>Matar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Filtro:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnComandos</sender>
   <signal>clicked()</signal>
   <receiver>GerenciadorWindow</receiver>
   <slot>enviaComando()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMatar</sender>
   <signal>clicked()</signal>
   <receiver>GerenciadorWindow</receiver>
   <slot>mataProcesso()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnContinuar</sender>
   <signal>clicked()</signal>
   <receiver>GerenciadorWindow</receiver>
   <slot>continuaProcesso()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnParar</sender>
   <signal>clicked()</signal>
   <receiver>GerenciadorWindow</receiver>
   <slot>pausaProcesso()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enviaComando()</slot>
  <slot>mataProcesso()</slot>
  <slot>continuaProcesso()</slot>
  <slot>pausaProcesso()</slot>
  <slot>atualizaView()</slot>
 </slots>
</ui>
